### ISSUE TITLE
Energy Sensor kit does not work on RF based machines

### ISSUE THREAD

**Issue by SteelShot** [xbony2/Nuclear-Control#155]
Well basically, when i use an energy sensor kit on for example the Resonant cell from TE, the kit gives me back the normal Energy Sensor location card, but it should give me the RF Energy Sensor Card.

Placing it in the Info Panel shows only that it has 32,000 energy.
![2016-05-30_23 45 14](https://cloud.githubusercontent.com/assets/5291010/15659316/a1b2d070-26c0-11e6-9638-1fc4f59f713a.png)

If i spawn the RF card, and place any RF powered block in 0,0,0 location it works fine and shows the correct information in this case a fully charged Resonant Cell.
![2016-05-30_23 46 52](https://cloud.githubusercontent.com/assets/5291010/15659327/c7edcff6-26c0-11e6-9c56-e9d74754b44f.png)
![2016-05-30_23 47 44](https://cloud.githubusercontent.com/assets/5291010/15659347/0361920c-26c1-11e6-881e-49fc39f68837.png)

Discharged some power from it.
![2016-05-30_23 48 07](https://cloud.githubusercontent.com/assets/5291010/15659353/18d0e426-26c1-11e6-9dc5-b0fd30ee23d3.png)
![2016-05-30_23 48 16](https://cloud.githubusercontent.com/assets/5291010/15659354/1bef94b8-26c1-11e6-8fcf-8faee7b6dfd7.png)

**Comment by xbony2** [comment]
@SteelShot mod list?

**Comment by SteelShot** [comment]
uhm gimme a min

**Comment by SteelShot** [comment]
http://modlister.com/user/SteelShot

**Comment by xbony2** [comment]
Oh boy.

**Comment by SteelShot** [comment]
Bad on my side, or yours? using the kit on mekanism works fine, tested that out few mins ago

**Comment by xbony2** [comment]
Nah, just a lot of mods. I'm not sure if this is a crossmod conflict or not.

**Comment by SteelShot** [comment]
Which mods should i try alone, IC2, Nuclear Conrol and just TE?

**Comment by xbony2** [comment]
Try using those alone ^_^

**Comment by SteelShot** [comment]
Well friend, something aint right here, same issue, IC2 is fine
![2016-05-31_00 35 59](https://cloud.githubusercontent.com/assets/5291010/15659724/c966a374-26c7-11e6-88d2-b007928a9ded.png)

Mods used in this run 
http://i.imgur.com/cVa7ieK.png

**Comment by SteelShot** [comment]
![2016-05-31_00 38 28](https://cloud.githubusercontent.com/assets/5291010/15659731/fef5c128-26c7-11e6-92a7-12bb653edff4.png)

**Comment by xbony2** [comment]
Alright, it's broken xD

**Comment by SteelShot** [comment]
If you need me to test test builds, im up for it, for the next 5 hrs or so

**Comment by xbony2** [comment]
Nah, I can pretty easily test it myself as required.

**Comment by SteelShot** [comment]
cool, just saying, i just got time and a server free :)

**Comment by dmf444** [comment]
Can you test with Mekanism? I have a feeling I introduced a bug while adding integration for that.

**Comment by SteelShot** [comment]
sure thing 3ish mins

**Comment by SteelShot** [comment]
Seems fine here
![2016-05-31_00 52 47](https://cloud.githubusercontent.com/assets/5291010/15659851/02dd4cfa-26ca-11e6-98a0-644baed8a100.png)

**Comment by SteelShot** [comment]
Btw i am using mek 9.0.2

**Comment by dmf444** [comment]
Yep. Then this bug is something that was introduced with the new mechanism stuff. I'll see if I can fix it. Thanks for helping us test!

**Comment by SteelShot** [comment]
Dont want to be rude or asking for a release, but is there any news on the issue? NBT editing doesnt seem to work all the time for me, something is messing it up

**Comment by dmf444** [comment]
The issue lies with the code itself. No number of NBT hacking will fix it. Your being given the wrong card, I just haven't had the time to fix it. Hopefully this weekend I'll try to force a patch in, else I just straight up revert the Mek integration. 
Tl;dr I'll try for a release this weekend

**Comment by SteelShot** [comment]
There is a work around with nbt editing tho, spawn in the RF card and add the block coordinate tags, i managed to do it once after multiple tries,  NBTedit crashes too much for me

**Comment by dmf444** [comment]
Fixed in recent commit. @xbony2 will release.

**Comment by xbony2** [comment]
I would, but the build has failed. I think Forge's servers are a bit messed up or something.
